babel.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
Was a babel config not necessary before?

Reply:
We didn't previously have one.